src/server/mod.rs

```diff
@@ -42,8 +42,10 @@ pub fn start(config: &str) {
     watcher.run(sender);
     loop {
         if let Ok(com) = receiver.recv() {
-            match com.message {
-                Message::Reload => state.reload(&watcher),
+            match com.message { Message::Reload => state.reload(&watcher),
+                Message::Start(task) if com.channel.is_some() => {
```

Comment:
I think `taskid` as var name is better, what do you think ?

Reply:
The tasks are launch by name, taskid would be a number for me, like pid or gid. "task" as a variable name works here and in the other files